jedi_language_server/server.py

```diff
@@ -421,7 +429,7 @@ def workspace_symbol(
     _symbols = (
         jedi_utils.lsp_symbol_information(name)
         for name in names
-        if name.module_path
+        if name.module_path is not None
         and str(name.module_path).startswith(workspace_root)
         and not _ignore_folder(str(name.module_path), ignore_folders)
     )
```

Comment:
Might just keep this one as-is given our weird additional checks that rely on `name.module_path` being non-`None`.